chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -215,6 +220,9 @@ def _apply_batch(self, batch: Batch) -> None:
         if (
             self._total_elements_added - self._persist_data.total_elements_added
             >= self._sync_threshold
+        ) or (
```

Comment:
what if the total elements added + updated > threshold

Reply:
whoops, good catch

Reply:
36f42187